bundle/NetgenBetterIbexaAdminUIBundle.php

```diff
@@ -4,8 +4,16 @@
 
 namespace Netgen\Bundle\BetterIbexaAdminUIBundle;
 
+use Netgen\Bundle\BetterIbexaAdminUIBundle\DependencyInjection\Compiler\SearchOverridePass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 final class NetgenBetterIbexaAdminUIBundle extends Bundle
 {
+    public function build(ContainerBuilder $container): void
+    {
+        parent::build($container);
```

Comment:
Parent method is empty, so no need for calling it.

Reply:
Fixed and amended.